test/fixtures/cwd.css

```diff
@@ -1,2 +1,2 @@
-@import "test/fixtures/imports/foo.css";
-@import "test/fixtures/imports/foo-recursive.css";
+@import "imports/foo.css";
+@import "imports/foo-recursive.css";
```

Comment:
it say "input", not "cwd" which is correct. If you make from a file `import "./stuff"`  and provide the from option, the file might be found in the same dir as the original input, and this cannot be done if you don't provide the from option.
What is not correctly clear ?
There is a test for that https://github.com/postcss/postcss-import/blob/debda623cd0d5354664b4412b87c29de1c16e6b6/test/index.js#L62